tools/mmp/driver.cs

```diff
@@ -771,12 +773,35 @@ static void Pack (IList<string> unprocessed)
 			if (App.LinkMode != LinkMode.All && App.RuntimeOptions != null)
 				App.RuntimeOptions.Write (App.AppDirectory);
 
+			if (experimental_xm_aot.HasValue && experimental_xm_aot.Value && IsUnified) {
+				AOTBundle ();
+				Watch ("AOT Compile", 1);
+			}
+
 			if (!string.IsNullOrEmpty (certificate_name)) {
 				CodeSign ();
 				Watch ("Code Sign", 1);
 			}
 		}
 
+		static void AOTBundle ()
+		{
+			string monoExe = Is64Bit ? "/usr/local/bin/mono64" : "/usr/local/bin/mono";
```

Comment:
hmm... not sure `mono32` can AOT 64bits code.

I think that XM needs to ship it's own AOT compilers since the system mono might not match with the BCL version XM ships.

IOW the above code should be fine when using the *system* mono (as long as both 32 and 64 bits are shipping)  but it might fail for the bundle 4.5 support coming out of XM.

For a first preview it might be good enough to specify which mono version(s) is/are compatible.